packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx

```diff
@@ -88,18 +97,47 @@ const ExpandedMobileWrapper = styled.div`
     display: flex;
     position: absolute;
     flex-direction: column;
-    background: ${({ theme }) => theme.BG_WHITE};
-    z-index: ${zIndices.expandableNavigation};
+    background: ${({ theme }) => theme.backgroundSurfaceElevationNegative};
     width: 100%;
     max-height: 80vh;
     overflow-y: auto;
     box-shadow: 0 4px 10px 0 ${({ theme }) => theme.BOX_SHADOW_BLACK_20};
     border-bottom-left-radius: 4px;
     border-bottom-right-radius: 4px;
-    padding: 0 16px;
     padding-bottom: 16px;
 `;
 
+const GradientBase = styled.div`
```

Comment:
nit: since the actual styled `div` is not needed, use this approach for when you have a base style and then simply spread it in `GradientBefore` and `After` ;)

```suggestion
const gradientBase = css`
```